src/components/FormRenderer/components/FormTemplate/index.tsx

```diff
@@ -26,20 +26,23 @@ export interface FormTemplateProps {
     submitLabel?: string;
 }
 
-const FormTemplate: FunctionComponent<FormTemplateProps> = ({
-    formFields,
-    schema: { cancelLabel = 'Cancel', submitLabel = 'Submit' },
-    schema,
-}) => {
+export interface AdditionalFormTemplateProps {
```

Comment:
Why not just update the FormTemplateProps?

Reply:
The FormTemplateProps are managed by data driven forms itself as we pass in the component class/function rather than a component instance:

```tsx
<FormRender
    componentMapper={{ ...componentMapper, ...customComponentWrapper }}
    FormTemplate={FormTemplate} // <-- here
    schema={schema}
    onSubmit={onSubmit}
    onCancel={onCancel}
    initialValues={initialValues}
/>
```

So I think the only way to pass extra stuff to it is to wrap it in another function so it's included in the closure :)

Reply:
The FormTemplateProps is on line 22. 

`export interface FormTemplateProps {
    formFields: any;
    schema: any;
    cancelLabel?: string;
    submitLabel?: string;
}`

You can add the flag there

Reply:
I feel like we should keep the implementation of the component clean.

Maybe wrap the component inside the FormRenderer something like 

```
const WrappedFormTemplate = useMemo((props: any) => (<FormTemplate {...props} isSubmitting></FormTemplate>), [isSubmitting]);

    return (
        <FormRender
            componentMapper={{ ...componentMapper, ...customComponentWrapper }}
            FormTemplate={WrappedFormTemplate}
            schema={schema}
            onSubmit={onSubmit}
            onCancel={onCancel}
            initialValues={initialValues}
        />
    );
```

Reply:
Much neater, thanks! Will update :)